app/views/account/users/_user.html.slim

```diff
@@ -0,0 +1,11 @@
+tr
+  td= link_to user.email, user.email
+  td= user.name
+  td
+    - if current_user.admin?
+      = simple_form_for(user, url: account_user_path(user)) do |f|
```

Comment:
form should be in TD

and again ... we are coming closer to the point when "edit user" form is needed instead of index page

speak with Kostya about implementing it